.circleci/config.yml

```diff
@@ -241,7 +241,7 @@ jobs:
           command: make license
       - run:
           name: cargo clippy
-          command: cargo clippy -- -D warnings -A clippy::upper_case_acronyms
+          command: cargo clippy -- -D warnings -A clippy::upper_case_acronyms -A clippy::needless_collect -A clippy::enum_variant_names -A clippy::needless_borrow
```

Comment:
Look at this @ec2 - this makes clippy pass on stable and instead of fixing these warnings in a single PR, I would like to remove these warnings in subsequent PRs, so you can better review the changes, and if needed, easier revert in case something goes south.